taskproj/service/models.py

```diff
@@ -0,0 +1,8 @@
+from django.db import models
+
+class Service(models.Model):
+    sname=models.CharField(max_length=50)
```

Comment:
@Ali-Ad it is better not to add first char from model name in the fields name instead of `sname` it should be `name` and the same for all fields in the model